tt_llk_blackhole/common/inc/sfpu/ckernel_sfpu_shift.h

```diff
@@ -13,16 +13,17 @@ namespace ckernel
 namespace sfpu
 {
 
-template <bool APPROXIMATION_MODE, int ITERATIONS>
+template <bool APPROXIMATION_MODE, InstrModLoadStore INSTRUCTION_MODE = INT32, bool SIGN_MAGNITUDE_FORMAT = false, int ITERATIONS = 8>
```

Comment:
```suggestion
template <bool APPROXIMATION_MODE, int ITERATIONS, InstrModLoadStore INSTRUCTION_MODE = INT32, bool SIGN_MAGNITUDE_FORMAT = false>
```
This way we keep backward compatibility. Unless you really need to change the order for some other reason?